frontend/src/Landing.jsx

```diff
@@ -0,0 +1,31 @@
+
+import { Link } from "react-router-dom";
+
+function Landing() {
+    
+  
+    return (
+      <div className="main">
+
+        <div className="mt-48 w-min">
```

Comment:
What is this class name? It's a little confusing as to what component it corresponds to